Tools/machines/lxplus-cern/spack.yaml

```
@@ -0,0 +1,55 @@
+spack:
+# CUDA
+  definitions:
+  - cuda: []
+  - cuda: [cuda, blaspp +cuda]
+    when: env.get("SPACK_STACK_USE_CUDA", "") == "1"
+
+# basic Python dependencies for WarpX Python PICMI bindings
+  - python: []
+  - python: [python, py-pip]
+    when: env.get("SPACK_STACK_USE_PYTHON", "") == "1"
+  specs:
+  - ncurses+symlinks
+  - adios2
+  - blaspp
+  - ccache
+  - cmake
+  - fftw
+  - hdf5
+  - lapackpp
+  - mpi
+  - openpmd-api
+  - pkgconfig
+  - $cuda
+  - $python
+
+  packages:
+    all:
+      target: [x86_64]
+      compiler: [gcc@11.2.0]
+      variants: ~fortran
+      providers:
+        mpi: [openmpi]
+    openmpi:
+      externals:
+      - spec: "openmpi"
+        prefix: /cvmfs/sft.cern.ch/lcg/releases/openmpi/3.1.0-37032/x86_64-centos7-gcc11-opt/
+      buildable: False
```

Error message was:
```
Spec version is not concrete: openmpi
```

I forgot that we also need to specify the version:
```suggestion
    openmpi:
      version: [3.1.0]
      externals:
      - spec: "openmpi@3.1.0"
        prefix: /cvmfs/sft.cern.ch/lcg/releases/openmpi/3.1.0-37032/x86_64-centos7-gcc11-opt/
      buildable: False
```

I hope this GCC11 OpenMPI build is compatible with your GCC 9.2.0 selection for the rest of the compiled software stack, otherwise we need to change this.